issue_release_status.go

```diff
@@ -131,41 +131,22 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 
 	logger.Debug().Msgf("%s", botMessage)
 
-	/*
-		var path string
-		var resp httpinternal.ListPoliciesResponse
-		var client *httpinternal.Client
-		var serviceName string
-
-		params := url.Values{}
-		params.Add("service", serviceName)
-
-		path, err := client.URLWithQuery(path, params)
-		if err != nil {
-			return err
-		}
-
-		err = client.Do(http.MethodGet, path, nil, &resp)
-	*/
-
 	// Send PR comment
-
-	/*client, err := handler.NewInstallationClient(installationID)
+	client, err := handler.NewInstallationClient(installationID)
 	if err != nil {
-		return errors.Wrap(err, "installing client")
+		return errors.Wrap(err, "creating new github.Client")
 	}
 
 	repositoryOwner := repository.GetOwner().GetLogin()
 	repositoryName := repository.GetName()
 
-	msg := fmt.Sprintf("To the stars! :-)")
 	newComment := github.IssueComment{
-		Body: &msg,
+		Body: &botMessage,
 	}
 
 	if _, _, err := client.Issues.CreateComment(ctx, repositoryOwner, repositoryName, prNum, &newComment); err != nil {
```

Comment:
I would've expected something like `client.PullRequests.CreateComment` here?
It this a wrong assumption on my part?

Reply:
I implemented your suggested change and tested it. It shows that IssueComment and Issue is the correct terminology to use in Github api terms. The alternative, PullRequestComment, is a comment a reviewer makes on code in the pull request. Just identical to the one you just wrote which I'm now replying to.